src/tito/cli.py

```diff
@@ -332,6 +332,11 @@ def __init__(self):
                 help="Custom arguments specific to a particular builder."
                     " (key=value)")
 
+        self.parser.add_option("--quiet", dest="quiet", action="store_true",
+                help="Suppress output from the build process.")
+        self.parser.add_option("--verbose", dest="verbose", action="store_true",
+                help="Expose more output from the build process.")
```

Comment:
Need to add these to the manpages.

--verbose is a bit close to --debug, we should probably merge them at some point in the future but if we do I'd lean towards the new --verbose in any case.